plugins/flytekit-greatexpectations/flytekitplugins/great_expectations/schema.py

```diff
@@ -66,7 +66,7 @@ class GreatExpectationsType(object):
     """
 
     @classmethod
-    def config(cls) -> Tuple[Type, Type[GreatExpectationsFlyteConfig]]:
+    def config(cls) -> Tuple[Type[str], GreatExpectationsFlyteConfig]:
```

Comment:
Why can we not have `Type` rather than `Type[str]`? `Type` should accept a string, correct?

Reply:
Yup, you're right. Updated it, Thanks.